modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -147,18 +147,18 @@ first line of the `controlPlane` section must not. Although both sections
 currently define a single machine pool, it is possible that future versions
 of {product-title} will support defining multiple compute pools during
 installation. Only one control plane pool is used.
-<3> Whether to enable or disable simultaneous multithreading, or
-`hyperthreading`. By default, simultaneous multithreading is enabled
-to increase the performance of your machines' cores. You can disable it by
-setting the parameter value to `Disabled`. If you disable simultaneous
-multithreading in some cluster machines, you must disable it in all cluster
-machines.
+<3> Whether to enable or disable simultaneous multithreading (SMT), or `hyperthreading`. By default, SMT is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable SMT, you must disable it in all cluster machines; this includes both control plane and compute machines.
 +
 [IMPORTANT]
 ====
 If you disable simultaneous multithreading, ensure that your capacity planning
 accounts for the dramatically decreased machine performance.
 ====
++
+[NOTE]
+====
+You must ensure SMT is enabled in your BIOS hardware settings of your machines, if applicable; otherwise, SMT will not work.
+====
```

Comment:
Maybe this would be obvious to the audience. 
* I assume that this config file does not enable SMT in the BIOS? Does it need to be clear where the config is enabling SMT, or is my ignornace showing?
* I assume that enabling the BIOS SMT is applicable when enabling SMT via the config file?

Reply:
Yeah, to my understanding, the BIOS SMT enabling is relevant only when enabling SMT in the install-config. If SMT is disabled in the install-config, the BIOS SMT settings are irrelevant.

>Does it need to be clear where the config is enabling SMT

@pamoedom do you have any feedback on whether more clarity into where SMT is being enabled (based on the config file) would be relevant, since we've now introduced other SMT settings (BIOS)? I don't have enough knowledge on the internal workings of SMT to make a judgment call.

Reply:
Yes, I was checking the installer code, [here](https://github.com/openshift/installer/blob/master/pkg/asset/machines/machineconfig/hyperthreading.go#L22) is the part that disables the SMT via kernel parameter in case of having `hyperthreading: Disabled` present in the `install-config.yaml`

However, the BIOS setting needs to be enabled (if present) in order to make use of the `hyperthreading` capabilities for baremetal because there is no cloudprovider who you can request that.

Most of the hardware vendors should have this option enabled by default (specially for recent hardware) but having a note that mentions it is a good idea IMHO.

Best Regards.

Reply:
@codyhoag, one idea, how about merging both notes on the following manner instead?

~~~
If you disable simultaneous multithreading both via `install-config.yaml` or via BIOS settings, ensure that your capacity planning accounts for the dramatically decreased machine performance.
~~~